inference-contract/README.md

```diff
@@ -0,0 +1,38 @@
+as determined necessary by the original asset owner. 
+
+The token object, as well as the asset guardian service are deployed by the asset owner, as part of the deployment sequence. The asset owner in addition authors the model scoring script that is executed as part of the guardian service frontend. Note that the model owner may use the token object to control key parameters used by the scoring script, and thus ensuring tight binding between the token object contract and the model usage.  
+
+## Deploying the OpenVINO Model Server Backend for tests ##
+
+Download the OpenVINO docker image.
+
+```bash
+docker pull openvino/model_server:latest
+```
+
+Our tests are based on the [classication demo model](https://github.com/openvinotoolkit/model_server/tree/main/demos/image_classification) contained within the OpenVINO repository. Please use the following commands use to download the model and deploy the OpenVINO model server docker container loaded with the classification model. 
+
+```bash
+mkdir -p imageclassification/model/1
+wget -P imageclassification/model/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.bin
+wget -P imageclassification/model/1 https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/2/resnet50-binary-0001/FP32-INT1/resnet50-binary-0001.xml
+docker run -d -u $(id -u):$(id -g) -v $(pwd)/imageclassification/model/:/models/ -p 127.0.0.1:9000:9000 openvino/model_server:latest --model_path /models/ --model_name resnet --port 9000 --shape auto
+```
+
+The model server endpoint is `127.0.0.1:9000`. Please note that we want the model server to be accessible only from the Guardian Service front end. In our tests, we assume that the Guardian Service and the model server are hosted on the same machine. The token owner (in practice siutated on a remote node) submits inference capability requests to the Guardian Service. The Guardian Service verifies the capability request,  gets its processed via the model server backend, and sends the result back to the token owner.
```

Comment:
Could be worth mentioning that port 9000 is defined in `etc/guardian_service.toml` (and hence could also be here and there changed if necessary)?

BTW: force-pushing during reviews is not really ideal; made me loose some (still relevant) comments ;-)  [put differently fixup commits are nicer although i have to admit as github (still?) doesn't not handle auto-squash automatically it requires a bit more complication during merge ....

Reply:
this is my fault. sort of.
i wanted more details in the commits.

Reply:
> this is my fault. sort of.
> i wanted more details in the commits.

I guess you are referring to my BTW?  Thinking of it, i guess it's true that fixup makes it easy to push code changes, there doesn't seem to be an easy way (other than delaying it to the autosquash) of updating commit log messages ...